collections/_explanations/xia2-merging.md

```diff
@@ -0,0 +1,51 @@
+---
+name: Merging in xia2 — the concept of `WAVELENGTH`
+---
+
+# Merging reflection intensities in xia2 — when a `WAVELENGTH` is not a wavelength
+
+Many crystallographers who use DIALS to integrate their rotation data don't actually interact directly with the DIALS commands.
+Instead, they often use [xia2](https://xia2.github.io) to co-ordinate the different components of the DIALS pipeline automatically.
+Here, we explore the concept of `WAVELENGTH` in xia2 and discover that it doesn't necessarily mean what it says.
+In fact, it turns out simply to be a label denoting a group of sweeps that should be merged together.
```

Comment:
In here I think you need to be clear that the _default_ behaviour is as a user may expect, however this is not mandatory.

Reply:
See comment below about _explanation_ vs _tutorial_.  I don't want to overburden this short document with responsibility for completely documenting xia2 usage and development, muddling together advice for novices and developers alike.